resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -623,6 +623,7 @@ private[spark] object KubernetesSuite {
   val decomTestTag = Tag("decom")
   val rTestTag = Tag("r")
   val MinikubeTag = Tag("minikube")
+  val usernameTestTag = Tag("username")
```

Comment:
@Yikun A new tag is added, you can use it to exclude this test for `apache/spark-docker` GA